frontend/src/components.css

```
@@ -0,0 +1,2546 @@
+}
+
+.tt-sprint-drawer-key-link {
+  white-space: nowrap;
+}
+
+.tt-sprint-drawer-title-link {
+  display: inline-block;
+  color: var(--t1);
+  text-decoration: none;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)